nautobot/project-static/css/base.css

```diff
@@ -755,3 +755,57 @@ pre code.hljs {
 code.hljs {
     padding: 2px 4px; /* Same as Bootstrap 3 `code` padding */
 }
+
+/* Table or tile dropdown context menu option spacings and icon colors */
+.context-menu li i {
+    margin-right: 10px;
+}
+.context-menu a:focus i,
+.context-menu a:focus span,
+.context-menu a:hover i,
+.context-menu a:hover span {
+    color: inherit;
+}
+
+/* Tile component styles */
+.tiles {
+    display: grid;
+    grid-gap: 20px;
+    grid-template-columns: repeat(auto-fit, minmax(445px, 1fr));
```

Comment:
What's the significance of 445px here?

Reply:
`445px` is the default tile width specified in Figma design. The way template colums attribute is set here for the tile grid, it squeezes in a single row as many `445px` tiles as it can, and then stretches them equally to fill the remaining space.

To give it more context, `auto-fit` has to be paired with a specific value (either absolute or relative, but not `auto`), be it `445px` or whatever else, otherwise it is ignored by the browser. The `minmax(..., 1fr)` part makes tiles stretch, if it wasn't for `minmax`, they would always be of static width like this:
![tiles_static](https://github.com/nautobot/nautobot/assets/117445994/75ea67a9-a187-43a1-8688-df3e4d0b133f)

Reply:
This image looks better given the width of the tiles. If we should change default px, we shouldn't need to stick to what was in Figma.

Reply:
@jedelman8, the tiles are `445px` wide in the image above with disabled stretching, so the only difference is they do not try to fill the entire space available like they do in the image in the main description. If you like it better, stretching can be disabled. What do you think @lampwins?

Reply:
Can/should we be using Bootstrap's `row`/`col` grid system for the tiles instead?

Reply:
That's what I also considered initially but Bootstrap grid system won't keep the same height of tiles in each row and as a result column wrapping may be off in some cases (see attached screenshot). Without custom scripting, current tiles behavior and appearance are only possible with CSS grid as far as I'm aware.

![tiles-bootstrap](https://github.com/nautobot/nautobot/assets/117445994/fa8f5c99-ca38-43b2-82ff-2c46c4527c8c)

Reply:
I prefer the way it is now with the height adjusting to the size of the tile content as you demoed earlier in the week. No doubt we will iterate on this in the not so distant future, hopefully with upgraded UI technology more available to us than bootstrap 3.

As for the width, I get the point Jason is making, but I also want to pick a value which results in a reasonable amount of tiles horizontally, for a given standard window size. For me, that is no more than 4 across, so I feel what we have is sufficient. Again, perhaps in the future, we can iterate to make it more dynamic.